android/app/src/main/python/mmrelay/plugin_loader.py

```
@@ -0,0 +1,938 @@
+                    if line.strip() and not line.startswith("#")
+                ]
+
+            if requirements:
+                if in_pipx:
+                    # Use pipx inject for each requirement
+                    logger.info(
+                        f"Installing requirements for plugin {repo_name} with pipx inject"
+                    )
+                    for req in requirements:
+                        logger.info(f"Installing {req}")
+                        subprocess.check_call(["pipx", "inject", "mmrelay", req])
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)